4-projects/business_unit_2/dev/example_single_project.tf

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "example_single_project" {
+  source = "../../modules/single_project"
+
+  org_id                      = var.org_id
+  billing_account             = var.billing_account
+  impersonate_service_account = var.terraform_service_account
+  environment                 = "dev"
+
+  folder_id = var.parent_folder
+
+  # Metadata
+  project_prefix   = "sample-single-bu2-d"
```

Comment:
Please correct me if I am wrong but looking at the white-paper, shouldn't project names be prefixed with `prf-`?

Reply:
Nice catch.
Actually it is `prj-`.  
I think that in this case it should be `prj-bu2-d-sample-single`

Reply:
Yep, we should rename project with that naming convention